packages/flutter/test/material/user_accounts_drawer_header_test.dart

```diff
@@ -139,6 +173,28 @@ void main() {
     expect(transformWidget.transform.getRotation()[4], 1.0);
   });
 
+  testWidgets('UserAccountsDrawerHeader icon does not rotate for setState', (WidgetTester tester) async {
+    await pumpTestWidget(tester);
+    Transform transformWidget = tester.firstWidget(find.byType(Transform));
+
+    // Icon is right side up.
+    expect(transformWidget.transform.getRotation()[0], 1.0);
+    expect(transformWidget.transform.getRotation()[4], 1.0);
+
+    await tester.tap(find.byKey(gestureKey));
+    await tester.pump();
+    await tester.pump(const Duration(milliseconds: 10));
+    expect(tester.hasRunningAnimations, isFalse);
+
+    await tester.pumpAndSettle();
```

Comment:
then this call here would be to the `StateSetter`.

Reply:
done